Automattic/jetpack
# Actions: move Gardening action into its own external project

#### Changes proposed in this Pull Request:

Our Gardening was something that could only be used within the Jetpack project until now. While this works for us, it would be good if others could use those tools in their own projects too.

This PR:

﻿- moves the Gardening action into its own external project
- Point our workflow to the new action location
- Adds a changelog (as well as the changelogger tooling) for the initial public release.

**To do**

- [x] Allow disabling some of the tasks via workflow configuration.
- [x] Improve the readme to explain what the action does, and what is needed for it to work.

#### Jetpack product discussion

* Internal reference: p3btAN-1nR-p2

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Do you still see a comment added below this PR, and labels added by a GitHub action?
* Check the different runs of the action for this PR: https://github.com/Automattic/jetpack/actions?query=branch%3Aupdate%2Fgardening-action-extract+workflow%3AGardening
